src/user/header/style.js

```diff
@@ -0,0 +1,15 @@
+import { makeStyles } from '@material-ui/core'
+
+const title = {
+  textAlign: 'center',
+  fontSize: '2vw',
+}
+
+const paper = {
+  paddingTop: '1vh',
+  paddingBottom: '1vh',
+  textAlign: 'center',
+  fontSize: '1vw',
+}
+
+export const useStyle = makeStyles(() => ({ title, paper }))
```

Comment:
Let's break up these style files as well. In that way, we can have a single inner field per style (called `root`) and thusly provide the styles to our components using the `classes` instead of the `className` prop.

Reply:
Done.